cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,24 +1,25 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get(
-        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
-    ).click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
-
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("#SubmitLogin").click();
+import {MenuContentPage, ProductListPage, ShopingCartPage, LoginPage} from "../page/index";
+import {AddressPage, ShippingPage, PaymentPage} from "../page/index";
```

Comment:
you can rewrite this as:
```suggestion
import {
  MenuContentPage,
  ProductListPage,
  ShopingCartPage,
  LoginPage,
  AddressPage,
  ShippingPage,
  PaymentPage
} from "../page/index";
```